vent/menus/add_options.py

```diff
@@ -50,8 +51,6 @@ def on_ok(self):
         """
         Take the branch, commit, and build selection and add them as plugins
         """
-        self.parentApp.repo_value['versions'] = {}
-        self.parentApp.repo_value['build'] = {}
```

Comment:
I totally blanked here and realized that I had removed these. I had thought you were suggesting I remove them. My mistake! I'll re-add them.